source/documentation/administration_guide/topics/Cluster_General_Settings_Explained.adoc

```diff
@@ -24,6 +24,24 @@ On existing clusters, the management network can only be changed using the *Mana
 * *ppc64*: For IBM POWER CPU types.
 
 |*CPU Type* |The oldest CPU family in the cluster. For a list of CPU types, see link:{URL_downstream_virt_product_docs}planning_and_prerequisites_guide/index#CPU_Requirements_RHV_planning[CPU Requirements] in the _Planning and Prerequisites Guide_. You cannot change this cannot after creating the cluster without significant disruption. Set CPU type to the oldest CPU model in the cluster. Only features present in all models can be used. For both Intel and AMD CPU types, the listed CPU models are in logical order from the oldest to the newest.
+|*Chipset/Firmware Type* a|This setting is only available if the *CPU Architecture* of the cluster is set to *x86_64*. This setting specifies the chipset and firmware type. Options are:
+
+* *Auto Detect*: This setting automatically detects the chipset and firmware type.
+* *I440FX Chipset with BIOS*: Specifies the chipset to I440FX with a firmware type of legacy BIOS.
+* *Q35 Chipset with Legacy BIOS*: Specifies the Q35 chipset with a firmware type of legacy BIOS without UEFI (Default for clusters with compatibility version 4.4)
+* *Q35 Chipset with UEFI BIOS* Specifies the Q35 chipset with a firmware type of legacy BIOS with UEFI
+* *Q35 Chipset with SecureBoot* Specifies the Q35 chipset with a firmware type of UEFI with SecureBoot, which authenticates the digital signatures of the boot loader.
+
+For more information, see link:{URL_virt_product_docs}{URL_format}administration_guide/index#About_UEFI_Q35-cluster_opt_settings[UEFI and the Q35 chipset] in the _Administration Guide_.
+|*Change Existing VMs/Templates from 1440fx to Q35 Chipset with Bios* |Select this check box to change  existing workloads when the cluster's chipset changes from I440FX to Q35.
+|*FIPS Mode* a|The FIPS mode used by the cluster. All hosts in the cluster must run the FIPS mode you specify or they will become non-operational.
+
+* *Auto Detect*: This setting automatically detects whether FIPS mode is enabled or disabled. When *Auto Detect* is selected, the FIPS mode is determined by the first host up in the cluster.
+
+* *Disabled*: This setting disables FIPS mode on the cluster.
```

Comment:
I think here and below it should be "disables FIPS" rather than "disables FIPS mode"